### PR TITLE
Move common stylesheets from `handsontable.css` to `walkontable…

### PR DESCRIPTION
**Pull request by warpech**
### Context

This PR splits `handsontable.css` into `handsontable.css` and a new `walkontable.css`

The main reason is that the styles that directly affect Walkontable should be defined closer to Walkontable source and tests. In the past, Walkontable had a separate stylesheet that was only used in Walkontable tests and this has lead to:
- significant differences between HOT and WOT stylesheets due to cumbersome mainainance
- tests of WOT were not testing exactly how HOT is rendered
- tests in WOT were generally neglected compared to HOT. For example, when you looked at WOT tests in a web browser, they often looked visually broken because of missing classes

This change was discussed in https://handsoncode.slack.com/archives/C4PV4S33Q/p1571751648008000.

### How has this been tested?

All tests continue to work. I have also tested manually in Chrome/Linux and Firefox/Linux.

⚠️ **There is a significant chance of a regression, because now the some style rules are loaded in a different order than before.** I suggest that this PR goes into the next major release (8.0.0-rc).

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/handsontable/issues/6381
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: src/3rdparty/walkontable/css/walkontable.test.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+body > .handsontable {
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 1.3em;
+  font-size: 13px;
+}
+
+.testOverflowHidden {
+  overflow: hidden;
+}
```
[the remark this answers is not in the document]
Comment (warpech) [reply src/3rdparty/walkontable/css/walkontable.test.css]: Where are these suggestions coming from? Do we have some CSS style guide?

[the remark this answers is not in the document]
Comment (warpech) [reply src/3rdparty/walkontable/css/walkontable.test.css]: Done https://github.com/handsontable/handsontable/issues/6387

**Comment by wojciechczerniak** [comment]
Just a question. Since now the Walkontable has selectors that are starting with `.hansontable` class. This isn’t ideal. Internal module knows about the module that is using it. We consider this as acceptable for now?

**Comment by warpech** [comment]
Good question, however apparently Walkontable already has hardcoded usage of `.handsontable` on `master` branch, so this is not a new problem: https://github.com/handsontable/handsontable/blob/master/src/3rdparty/walkontable/src/table.js#L207

**Comment by wojciechczerniak** [comment]
> But maybe not in this PR?

Surely! I didn't notice `.handsontable` was already there. Refactoring this is a very different story then and should be done separately. 

> Should this PR be merged to `develop`? I think it should go in `8.0.0` branch, not in `7.x`.

I would consider this a breaking change. If someone tweaked Handsontable appearance the selectors might have different specificity now. +1 for `8.0.0`

👍

**Pull request merged**